jenkins/catapult/jenkins/catapult-client-build-base-image-all.groovy

```diff
@@ -141,6 +147,20 @@ pipeline {
 				}
 			}
 		}
+		post {
+			unsuccessful {
```

Comment:
should we publish on success too ?

Reply:
Would this be just for the image jobs?  The dailies jobs are too noisy imo